sticker2-utils/protos/sticker2.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+// Request for annotations.
+message AnnotateSentenceRequest {
+  repeated string tokens = 1;
+}
+
+// A response consists of either an error or an annotated sentence.
+message AnnotateSentenceResponse {
+  oneof result {
+    string error = 1;
```

Comment:
Maybe this should be a separate type as well, so that we can easily other fields to an error in the future?